modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -56,163 +52,166 @@ For example:
-.. Select "Upgrade to vx.x". The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
-.. Wait for the status to transition to *Active*. 
+=== Upgrade Runtime Fabric Components
+
+. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
+. Select *Upgrade to vx.x*. 
++
+The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
+. Wait for the status to transition to *Active*. 
 +
 In most cases, the upgrade takes fewer than 10 minutes. When the status transitions to *Active* and the version of the Runtime Fabric reflects the new version, the component upgrade is complete.
 +
 If the Runtime Fabric and Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
-+
-Keep your browser session open to use in later steps.
-+
-[IMPORTANT]
-To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, a button to upgrade to 1.4.54 is displayed. Upgrade to this version in order to enable upgrades to 1.5.
-. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version:
-.. Using your terminal on a controller node, run the following command: 
+. Keep your browser session open to use in later steps.
+
+=== Upgrade the `rtfctl` Command-line Utility
+
+. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version with the following command:
 +
 ----
 cd /opt/anypoint/runtimefabric
 sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
 ----
-+
-.. Change file permissions for the `rtfctl` binary: 
+. Change file permissions for the `rtfctl` binary: 
 +
 ----
 sudo chmod +x rtfctl
 ----
-. If you are not using the latest version of the appliance, upgrade to the latest version:
+. Alternatively, to update `rtfctl` in place as a user who owns the `rtfctl` binary, run:
++
+----
+`rtfctl update`
+----
 
-.. Make sure the `/opt/anypoint/runtimefabric/installer` directory has a minimum of 4 GiB of disk space available.
-.. Verify you have root privileges. Root privileges are required to perform an upgrade.
-.. Verify your environment meets the prerequisites:
+=== Upgrade the Runtime Fabric Appliance
 
-*** The internal load balancer must be running on at least three replicas to maintain availability. On newer versions of Runtime Fabric, the internal load balancer runs on all controller nodes by default.
-*** An external load balancer must be configured to load balance incoming requests to the controller nodes, with a health check configured for TCP port 443.
-*** The CPU and memory resources for at least one additional worker node must be available in the cluster.
+Before upgrading the Runtime Fabric appliance, verify the following: 
```

Comment:
So...these are actually prereqs.
You can either use the Prereq heading or just intro. things like this as you have here, with "Before <performing this task>, but try to standardize to one approach per deliverable.
(Note that future task template might dictate which of these we use going forward.)